bot/cogs/information.py

```diff
@@ -211,53 +223,85 @@ async def create_user_embed(self, ctx: Context, user: Member) -> Embed:
         # Custom status
         custom_status = ''
         for activity in user.activities:
-            # Check activity.state for None value if user has a custom status set
-            # This guards against a custom status with an emoji but no text, which will cause
-            # escape_markdown to raise an exception
-            # This can be reworked after a move to d.py 1.3.0+, which adds a CustomActivity class
-            if activity.name == 'Custom Status' and activity.state:
-                state = escape_markdown(activity.state)
-                custom_status = f'Status: {state}\n'
+            if isinstance(activity, CustomActivity):
+                state = ""
+
+                if activity.name:
+                    state = escape_markdown(activity.name)
+
+                emoji = ""
+                if activity.emoji:
+                    if not activity.emoji.id:
+                        emoji += activity.emoji.name + " "
```

Comment:
Is the `activity.emoji.id` check here to filter custom emoji? I think it could use an in-line comment. It's a little indirect.